.circleci/config.yml

```diff
@@ -102,6 +102,32 @@ jobs:
 
 
 
+  e2eTestCurPRSameClusterID:
+    environment:
+      TESTED_VERSION: "current"
+      TEST_DIR: "integration/test/sameclusterid"
```

Comment:
I think about renaming it to `integration/test/recreatecluster`. Is that a better name?

Reply:
I don't mind. Lets maybe just have some godoc in the go test to explain what this thing is doing.